.github/workflows/databend-release.yml

```diff
@@ -157,6 +163,8 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
maybe docker image build do not need this?

Reply:
oh, yes